application/account-management/WebApp/pages/admin/users/index.tsx

```diff
@@ -14,14 +14,14 @@ export default function UsersPage() {
   return (
     <div className="flex gap-4 h-screen bg-gray-50">
       <SideMenu />
-      <div className="flex flex-grow flex-col gap-4 pl-1 pr-6 py-3 overflow-x-auto">
-        <div className="z-10">
+      <div className="flex flex-grow flex-col gap-4 pl-1 pr-6 py-3 overflow-x-auto bg-gray-50">
+        <div className="z-10 min-w-[500px]">
```

Comment:
we should try to avoid custom styling and rely on design tokens

Reply:
I reverted this for now. It's just that the current design does not render correct on small screens, so showing this on an iPhone is not good. I know this was a temporary solution, as we should make a hamburger menu and create a true mobile friendly UI.